github all release + all assets total download count

sagiegurari opened the pull request:
I think the title says it all.
Basically it helps smaller projects with binary assets to show total downloads of all their releases together.

espadrine commented on index.html:
```diff
@@ -913,6 +917,10 @@ <h2 id="contributors"> Contributors </h2>
+<br>
+<a class='photo' href='https://github.com/sagiegurari'>
+  <img alt='sagiegurari' src='https://avatars.githubusercontent.com/u/8112599?s=80'>
+</a>
```
I think you want to put that in try.html, as index.html gets generated from it.

espadrine commented on server.js:
```diff
@@ -2648,6 +2648,51 @@ cache(function(data, match, sendBadge, request) {
+// GitHub all releases download count integration.
+camp.route(/^\/github\/dt\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var user = match[1];  // eg, qubyte/rubidium
```
Could you put

espadrine commented on server.js:
```diff
@@ -2648,6 +2648,51 @@ cache(function(data, match, sendBadge, request) {
 }));
 
+// GitHub all releases download count integration.
+camp.route(/^\/github\/dt\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```
Could you merge this with `github/downloads` please? We can consider that a tag named `total` actually means that we should look at all tags.

espadrine commented:
Thanks for contributing!

As I explain in the review, I would prefer to merge the URL for this with `/github/downloads/`.

Could you squash the commits together?

sagiegurari commented:
Thanks for the review.
I'll handle those changes tomorrow.

sagiegurari commented:
I updated based on your comments however, I wasn't 100% sure about github\/downloads consolidation, since the URL pattern is different in the current service and wasn't sure if it will catch shorter urls (with less params).
so there are now 2 services for the github\/downloads pattern but they have different params amount.
The original:
````js
camp.route(/^\/github\/downloads\/([^\/]+)\/([^\/]+)\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
````
The added (by me):
````js
camp.route(/^\/github\/downloads\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
````

There are also now merge conflicts, so do you need to me merge from your master again?
It was after I squashed my last 9 commits into 1.

sagiegurari commented:
due to all the conflicts I recreated the pull request with an up to date repo.
See #535
